test/apps/todo/lib/todo/commands/create.rb

```diff
@@ -1,53 +1,54 @@
-desc "Create a new task or context"
-command [:create,:new] do |c|
-  c.desc "Make a new task"
-  c.arg_name 'task_name', :multiple
-  c.arg :should_ignore_this
-  c.command :tasks do |tasks|
-    tasks.action do |global,options,args|
-      puts "#{args}"
+class App
```

Comment:
Does these all need to be inside `App`?  ideally `commands_from` will require this inside whatever context it's in and thus should not need to be wrapped (I think)

Reply:
Currently it does, because `commands_from` is a simple `require` [here](https://github.com/davetron5000/gli/blob/84c21131320168178077e9bfe03f51f0e28d361e/lib/gli/app.rb#L319). I opened https://github.com/davetron5000/gli/issues/279.

Reply:
D'oh, yeah.  I thought for some reason it worked differently.  I think this PR is fine as I don't want you to have address the issue with `commands_from` as well, since that's a bit tricky